sql-join/what-and-when.sql

```diff
@@ -0,0 +1,8 @@
+select "releaseYear" as "Release Year",
```

Comment:
Aliasing to `Release Year` here is okay but in the future we will be using SQL in our JavaScript so it is best to keep column names and aliases to valid variable name formats. This will make the data easier to work with in our code.